### ISSUE TITLE
Bug: formatting outside of docstring when first word is "default"

### ISSUE THREAD

**Issue by jamesbraza** [PyCQA/docformatter#165]
Pretty niche bug here, `docformatter` is formatting outside of the docstring:

```python
def foo():
    default = 0  # Comment

    def bar():
        pass
```

Running `docformatter==1.5.1` on this:

```console
> docformatter a.py
--- before/a.py
+++ after/a.py
@@ -1,5 +1,4 @@
 def foo():
     default = 0  # Comment
-
     def bar():
         pass
```

**Comment by weibullguy** [comment]
This is the same as the issue in #156 where the blank line is removed when there is a comment following the last line in a code block.  Closing as a duplicate to #156.

**Comment by jamesbraza** [comment]
Okay sounds good, didn't know they were the same issue.

@weibullguy one thing to note, is changing the variable name in the first line from `default` to something like `ham`, the issue goes away:

```python
def foo():
    ham = 0  # Comment

    def bar():
        pass
```

`docformatter` seems to be matching the variable name, if that helps

**Comment by weibullguy** [comment]
@jamesbraza Thanks for the additional information.  It's weird behavior for sure :confused:  But all the information I can get is helpful.

**Comment by weibullguy** [comment]
@jamesbraza tag v1.6.1-rc1 should have the fix you need if you're interested in giving it a try before I officially release v1.6.1.

**Comment by jamesbraza** [comment]
I wasn't sure how to `pip install` the [v1.6.1-rc1](https://github.com/PyCQA/docformatter/releases/tag/v1.6.1-rc1), I tried `pip install --pre` and got nothing.  I went ahead `git clone`'d the repo, and confirmed that this issue is resolved with the current `master` commit [f1b0fa688a](https://github.com/PyCQA/docformatter/tree/f1b0fa688af908cfe9d8e56420660dc1cd2daf7e).

Nice job, thank you! ✅
